### DISCUSSION TITLE
https://broadenmeritdainty.best/viewer/sk/194/5/2787325715/u/1427224/0/

### DISCUSSION THREAD

**Discussion by EpickyHrac** [sk-cat/fluffy-blocklist#10] [Domains to add] [answered]
<img src="https://preco-a-ako-si-ziskal.c0ronavir.us/0aB6Cbda7dc0Aa2C.png"></img>
<img src="https://preco-a-ako-si-ziskal.c0ronavir.us/FFaF4D5b7fDF87FA.png"></img>
——————————————————————————————

——————————————————————————————
**WHO IS**
NetRange:       172.64.0.0 - 172.71.255.255
CIDR:           172.64.0.0/13
NetName:        CLOUDFLARENET
NetHandle:      NET-172-64-0-0-1
Parent:         NET172 (NET-172-0-0-0-0)
NetType:        Direct Assignment
OriginAS:       AS13335
Organization:   Cloudflare, Inc. (CLOUD14)
RegDate:        2015-02-25
Updated:        2017-02-17
Comment:        All Cloudflare abuse reporting can be done via 
https://www.cloudflare.com/abuse
Ref:            https://rdap.arin.net/registry/ip/172.64.0.0

OrgName:        Cloudflare, Inc.
OrgId:          CLOUD14
Address:        101 Townsend Street
City:           San Francisco
StateProv:      CA
PostalCode:     94107
Country:        US
RegDate:        2010-07-09
Updated:        2021-01-11
Ref:            https://rdap.arin.net/registry/entity/CLOUD14

OrgAbuseHandle: ABUSE2916-ARIN
OrgAbuseName:   Abuse
OrgAbusePhone:  +1-650-319-8930 
OrgAbuseEmail:  
OrgAbuseRef:    https://rdap.arin.net/registry/entity/ABUSE2916-ARIN

OrgNOCHandle: NOC11962-ARIN
OrgNOCName:   NOC
OrgNOCPhone:  +1-650-319-8930 
OrgNOCEmail:  
OrgNOCRef:    https://rdap.arin.net/registry/entity/NOC11962-ARIN

OrgTechHandle: ADMIN2521-ARIN
OrgTechName:   Admin
OrgTechPhone:  +1-650-319-8930 
OrgTechEmail:  
OrgTechRef:    https://rdap.arin.net/registry/entity/ADMIN2521-ARIN

RNOCHandle: NOC11962-ARIN
RNOCName:   NOC
RNOCPhone:  +1-650-319-8930 
RNOCEmail:  
RNOCRef:    https://rdap.arin.net/registry/entity/NOC11962-ARIN

RTechHandle: ADMIN2521-ARIN
RTechName:   Admin
RTechPhone:  +1-650-319-8930 
RTechEmail:  
RTechRef:    https://rdap.arin.net/registry/entity/ADMIN2521-ARIN

RAbuseHandle: ABUSE2916-ARIN
RAbuseName:   Abuse
RAbusePhone:  +1-650-319-8930 
RAbuseEmail:  
RAbuseRef:    https://rdap.arin.net/registry/entity/ABUSE2916-ARIN

**Comment by 0Charliecat** [comment, marked as answer]
Added [sk-cat/fluffy-blocklist/phisising/Facebook.txt](https://github.com/sk-cat/fluffy-blocklist/blob/main/phisising/Facebook.txt)

**Reply by 0Charliecat** [reply]
[sk-cat/fluffy-blocklist/domains](https://github.com/sk-cat/fluffy-blocklist/blob/main/domains)
